circle.yml

```diff
@@ -13,6 +13,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+machine:
+  java:
+    version: openjdk7
```

Comment:
can we change this to openjdk8? Notice that current parent jboss pom specifies java 8 as version.